lib/token_store.rb

```diff
@@ -1,17 +1,44 @@
 class TokenStore
+  cattr_reader :token_caches
+
   @token_caches = {} # Hash of Memory/Dalli Store Caches, Keyed by namespace
 
+  module KeyValuehelpers
```

Comment:
Capital H?

Reply:
Oops!  Good catch.

Reply:
Heh, I didn't realize I was consistent with all of my usages of this.  No wonder I didn't notice it being a syntax error or something... 😅